api/src/main/java/com/epam/pipeline/entity/cluster/NodeInstance.java

```diff
@@ -45,7 +45,6 @@
 public class NodeInstance extends AbstractSecuredEntity {
 
     private UUID uid;
-    private String name;
```

Comment:
I'm afraid that this change can be harmful if some api method has `NodeInstance` class in request body. For example as part of `PipelineRun` class. Let's revert the change until we have implemented tests for all the controllers.